packages/rlc-common/src/metadata/buildVitestConfig.ts

```diff
@@ -0,0 +1,118 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { Project } from "ts-morph";
+import { RLCModel } from "../interfaces.js";
+
+const nodeConfig = `export default defineConfig({
+  test: {
+    "reporters": ["basic", "junit"],
+    "outputFile": {
+      "junit": "test-results.browser.xml"
+    },
+    "fakeTimers": {
+      "toFake": ["setTimeout", "Date"]
+    },
+    "watch": false,
+    "include": ["test/**/*.spec.ts"],
+    "exclude": ["test/**/browser/*.spec.ts"],
+    "coverage": {
+      "include": ["src/**/*.ts"],
+      "exclude": [
+        "src/**/*-browser.mts",
+        "src/**/*-react-native.mts",
+        "vitest*.config.ts",
+        "samples-dev/**/*.ts"
+      ],
+      "provider": "istanbul",
+      "reporter": ["text", "json", "html"],
+      "reportsDirectory": "coverage"
+    }
+  }
+});`;
+
+const browserConfig = (options: {
+  isAzureSdkForJs: boolean;
+}) => `export default defineConfig({
+    "define": {
+      "process.env": process.env
+    },
+    ${options.isAzureSdkForJs ? `plugins: [browserMap()],` : ""}
+    "test": {
+      "reporters": ["basic", "junit"],
+      "outputFile": {
+        "junit": "test-results.browser.xml"
+      },
+      "browser": {
+        "enabled": true,
+        "headless": true,
+        "name": "chromium",
+        "provider": "playwright"
+      },
+      "fakeTimers": {
+        "toFake": ["setTimeout", "Date"]
+      },
+      "watch": false,
+      "include": ${
+        options.isAzureSdkForJs
+          ? `["dist-test/browser/**/*.spec.js"]`
+          : `["test/**/*.spec.ts"]`
```

Comment:
i may not get the point that we need to distugsh with `isAzureSdkForJs` option with diff path?